### PR TITLE
Update Community Call in the README

### PR DESCRIPTION
**Pull request by DhritiShikhar**
### Motivation

Community calls have been updated, add new meeting link in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,9 +143,10 @@ The following example scenarios are available:
 
 ## Community, discussion, contribution, and support
 
-The Service Binding community meets weekly on Thursdays at 11:15 AM UTC via [Google Meet](https://meet.google.com/jti-pujt-zqs).
+The Service Binding community meets weekly on Thursdays at 1:00 PM UTC via [Google Meet](meet.google.com/hxg-ooyk-ucx
+).
 
-Meeting Agenda is maintained [here](https://github.com/redhat-developer/service-binding-operator/issues?q=is%3Aissue+is%3Aopen+label%3Ameeting)
+Meeting Agenda is maintained [here](https://www.google.com/url?q=https://docs.google.com/document/d/1LLyqe9u4_JooVUB35bttY4QxYHaI6leewEFfxgzFp2E/edit?usp%3Dsharing&sa=D&source=calendar&ust=1610451660967000&usg=AOvVaw0uC4tppzktn7JwDiwW-nNM)
```
Comment (jasperchui) [inline README.md:149]: Right now we have configured a google group to manage access.  Please update to include instructions to request for access by sending an email to our service-binding-operator@redhat.com email address.  Thanks.

**Review by jasperchui — CHANGES_REQUESTED** [review]
Requested clarification update to request access to minutes/agenda.

File: README.md (hunk 1 above)
Comment (pedjak) [reply README.md:149]: this looks are very high barrier, just to be able to check agenda... IMHO, we should create a document that is viewable for everybody and then give appropriate rights to people who can edit its content. For example, Helm is doing this properly: https://github.com/helm/community/blob/master/communication.md#meetings

Comment (DhritiShikhar) [reply README.md:149]: Agreed @pedjak 
Everybody should be able to read the agenda.
Thanks for sharing the helm link. I would update the README accordingly. 

@jasperchui Currently the agenda is not readable for everyone.
I logged in from my personal gmail account and was not able to read the doc.
Can we change that?

Comment (jasperchui) [reply README.md:149]: @DhritiShikhar could you update the meeting notes to this: https://docs.google.com/document/d/1HwhAKqpM6l4Ur3h3IApDFzbH2Y_xvj_n1x1pEdwRuSY/edit?usp=sharing
this should have permission for anyone to view. Thanks.

Comment (DhritiShikhar) [reply README.md]: Done @jasperchui Please review again

**Comment by DhritiShikhar** [comment]
@jasperchui @pedjak Please review again

**Comment by DhritiShikhar** [comment]
Merging the PR as suggested by @jasperchui

**Pull request merged**